tests/virt_autotest/install_package.pm

```diff
@@ -23,8 +24,14 @@ sub install_package {
         set_var('QA_HEAD_REPO', $qa_server_repo);
         bmwqemu::save_vars();
     }
-    script_run "zypper --non-interactive rr server-repo";
-    assert_script_run("zypper --non-interactive --no-gpg-check ar -f '$qa_server_repo' server-repo");
+    if (check_var('ARCH', 's390x')) {
+        lpar_cmd("zypper --non-interactive rr server-repo");
+        lpar_cmd("zypper --non-interactive --no-gpg-check ar -f '$qa_server_repo' server-repo");
+    }
+    else {
+        script_run "zypper --non-interactive rr server-repo";
+        assert_script_run("zypper --non-interactive --no-gpg-check ar -f '$qa_server_repo' server-repo");
```

Comment:
On several occasions the actual executed code - e.g. `zypper --non-interactive rr server-repo` - is the same but is run via a different wrapper function - i.e. `lpar_cmd()` or `{assert,}script_run()` - based on result of `check_var('ARCH', 's390x')`. I think this should be unified to a single wrapper function, which prevents the code duplication.

Reply:
Good point, we had discussed this as your mentioned before summit this PR. 
Moreover, this PR will not cover this as your mentioned, imho.
And also, we will summit new PR to fix this.

Reply:
Fine by me, thanks.